hugegraph-core/src/main/java/com/baidu/hugegraph/backend/cache/AbstractCache.java

```diff
@@ -74,17 +83,19 @@ public V get(K id) {
             value = this.access(id);
         }
 
-        if (value == null) {
-            ++this.miss;
-            if (LOG.isDebugEnabled()) {
-                LOG.debug("Cache missed '{}' (miss={}, hits={})",
-                          id, this.miss, this.hits);
-            }
-        } else {
-            ++this.hits;
-            if (LOG.isDebugEnabled()) {
-                LOG.debug("Cache cached '{}' (hits={}, miss={})",
-                          id, this.hits, this.miss);
+        if (this.enabledMetrics) {
+            if (value == null) {
+                this.miss.add(1L);
+                if (LOG.isDebugEnabled()) {
+                    LOG.debug("Cache missed '{}' (miss={}, hits={})",
+                              id, this.miss, this.hits);
+                }
+            } else {
+                this.hits.add(1L);
+                if (LOG.isDebugEnabled()) {
+                    LOG.debug("Cache cached '{}' (hits={}, miss={})",
+                              id, this.hits, this.miss);
+                }
```

Comment:
I construct it for a method named `metricCacheIfNeed()` to make `get()` simplified (in another pr)